examples/uspto/README.md

```diff
@@ -0,0 +1,5 @@
+# Applicants Population by Year and Country
+
+The graph illustrates the calculated population of applicants who are listed in the records of the US Patent and Trademark Office, focusing specifically on the top five countries of the year 2022.
```

Comment:
The chart illustrates the number of applicants listed in the records of the US Patent and Trademark Office,
tracking those from the top five countries of the year 2022.